napari_console/qt_console.py

```diff
@@ -161,9 +161,7 @@ def _update_theme(self, event=None):
         # (should probably be done by napari)
         # After napari 0.4.11, themes are evented models rather than
         # dicts.
-        with warnings.catch_warnings():
-            warnings.simplefilter("ignore", FutureWarning)
-            theme = get_theme(self.viewer.theme)
+        theme = get_theme(self.viewer.theme, as_dict=True)
```

Comment:
This is the reason for this full PR, `theme = get_theme(self.viewer.theme)` is deprecated, and replaced by `get_theme(self.viewer.theme, as_dict=True)`, once this is done we can remove the deprecated API from napri.